sycl/source/detail/global_handler.cpp

```diff
@@ -71,6 +71,49 @@ std::atomic_uint ObjectUsageCounter::MCounter{0};
 GlobalHandler::GlobalHandler() = default;
 GlobalHandler::~GlobalHandler() = default;
 
+void GlobalHandler::InitXPTIStuff() {
+#ifdef XPTI_ENABLE_INSTRUMENTATION
+  // Let subscribers know a new stream is being initialized
+  getXPTIRegistry().initializeStream(SYCL_STREAM_NAME, GMajVer, GMinVer,
+                                     GVerStr);
+  xpti::payload_t SYCLPayload("SYCL Runtime Exceptions");
+  uint64_t SYCLInstanceNo;
+  GSYCLCallEvent = xptiMakeEvent("SYCL Try-catch Exceptions", &SYCLPayload,
+                                 xpti::trace_algorithm_event, xpti_at::active,
+                                 &SYCLInstanceNo);
+#endif
+}
+
+void GlobalHandler::TraceEventXPTI(const char *Message) {
+  if (!Message)
+    return;
+#ifdef XPTI_ENABLE_INSTRUMENTATION
```

Comment:
```suggestion
#ifdef XPTI_ENABLE_INSTRUMENTATION
  if (!Message)
    return;
```

This code has no value if `XPTI_ENABLE_INSTRUMENTATION` is not set.

Reply:
done